src/vsr/superblock.zig

```diff
@@ -85,29 +78,28 @@ pub const SuperBlockHeader = extern struct {
     /// The checksum of the previous superblock to hash chain across sequence numbers.
     parent: u128,
 
-    /// The checksum over the actual encoded block free set in the superblock trailer.
-    free_set_checksum: u128,
```

Comment:
While this is not strictly necessary, I think I'll add back the checksum covering the entire free set, as it can protect from encoding/decoding bugs. I think I'll leave it to a follow up though.

Reply:
Fantastic idea to verify the roundtrip.